src/lammpsio/data.py

```diff
@@ -394,6 +404,18 @@ def read(self):
                                 "Expected number of columns not read for mass"
                             )
                         masses[int(row[0])] = float(row[1])
+                elif "Bonds" in line:
+                    _readline(f, True)  # blank line
+                    bonds.typeid = numpy.zeros(bonds.N)
+                    bonds.members = numpy.zeros([bonds.N, 2])
```

Comment:
I don't think you will need this if you implemented the properties correctly in `Topology` to self-allocate.
```suggestion
```